### PR TITLE
fixed vote consumption

### PR DESCRIPTION
**Pull request by maikelmclauflin**
* flip a bool passed to the consume function to have sinek not convert to string first so that we don’t get bad number deserialization errors. (`80` -> `-127`) 5f282d2
* isolate kafka topics so that if one goes down it doesn't prevent the others from continuing f3524a6

### CODE REVIEW

File: bat-utils/lib/runtime-kafka.test.js [hunk 1]
```diff
@@ -2,28 +2,124 @@
...
 
-  t.is(messages.length, 1)
-  t.is(messages[0].value, 'hello world')
+  const consumers = await consumer.consume()
+  const messages = []
+  for (let i = 0; i < 10; i += 1) {
+    messages.push(sendMsgs())
+  }
+  const expectingTopic1 = [].concat.apply([], await Promise.all(messages))
+  await waitForParody(topic1)
```
Comment (evq) [inline bat-utils/lib/runtime-kafka.test.js:72]: *parity

File: bat-utils/lib/runtime-kafka.test.js [hunk 2]
```diff
@@ -2,28 +2,124 @@
...
+  // check state
+  t.true(state[topic1].length === 100, 'topic 1 should have processed 100 msgs')
+  t.deepEqual(expectingTopic1, state[topic1], 'topic 1 state should be as expected')
+  const expectedLength = consumptionPattern.reduce((memo, value) => {
+    return (memo + value) > errAt ? memo : (memo + value)
+  })
+  console.log('consumption pattern', expectedLength, consumptionPattern)
+  t.is(expectedLength, state[topic2].length, `topic 2 should be less than or equal to ${expectedLength} in length`)
+  const expectedStateTopic2 = expectingTopic1.slice(0, state[topic2].length)
+  t.deepEqual(expectedStateTopic2, state[topic2], 'topic2 should the frist ordered subset of topic 1')
```
Comment (evq) [inline bat-utils/lib/runtime-kafka.test.js:83]: *first

File: bat-utils/lib/runtime-kafka.test.js (hunk 1 above)
Comment (maikelmclauflin) [reply bat-utils/lib/runtime-kafka.test.js]: fixed: 6280db6

File: bat-utils/lib/runtime-kafka.test.js (hunk 2 above)
Comment (maikelmclauflin) [reply bat-utils/lib/runtime-kafka.test.js]: fixed: 6280db6

**Pull request merged**